### ISSUE TITLE
Difficulty locating .jar file download

### ISSUE THREAD

**Issue by merlinlikethewizard** [miho/JCSG#63]
[Disclaimer, I may have only had trouble because I'm dumb]

The .jar download link was a bit hard for me to find. There's the very small link to bintray from the README, then it says "No direct downloads selected for this package," so you have to click "Files," then it looks like there's nothing there until you see the quite tiny "0.5.7" folder, then the right download is in the list with a bunch of other stuff... maybe I'm just dumb but I wasted a good amount of time thinking there was no .jar download and trying to figure out how to use Gradle with the source code to try and make something importable to my project.

TL;DR Please someone smarter than me make the link easier to find.
